### PR TITLE
Update for Swift 4.2

### PR DESCRIPTION
**Pull request by keith**

### CODE REVIEW

**Comment by keith** [comment]
We shouldn't merge until the GM

File: Sources/LambdaKit/UIImagePickerController+LambdaKit.swift [hunk 1]
```diff
@@ -78,8 +78,9 @@ extension UIImagePickerController: UIImagePickerControllerDelegate, UINavigation
 
     // MARK: UIImagePickerControllerDelegate implementation
 
-    open func imagePickerController(_ picker: UIImagePickerController,
-        didFinishPickingMediaWithInfo info: [String : Any])
+    open func imagePickerController(
+        _ picker: UIImagePickerController,
+        didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any])
```
Comment (rebello95) [inline Sources/LambdaKit/UIImagePickerController+LambdaKit.swift]: `[UIImagePickerController.InfoKey: Any]`

**Review by rebello95 — APPROVED** [review]

File: Sources/LambdaKit/UIImagePickerController+LambdaKit.swift (hunk 1 above)
Comment (keith) [reply Sources/LambdaKit/UIImagePickerController+LambdaKit.swift]: That can't be inferred

Comment (rebello95) [reply Sources/LambdaKit/UIImagePickerController+LambdaKit.swift]: Sorry, just meant the spacing, not the type name

Comment (keith) [reply Sources/LambdaKit/UIImagePickerController+LambdaKit.swift]: ah yea, migrator

Comment (keith) [reply Sources/LambdaKit/UIImagePickerController+LambdaKit.swift]: oh actually it was wrong before

File: Sources/LambdaKit/UIWebView+LambdaKit.swift [hunk 2]
```diff
@@ -25,7 +25,7 @@
 import Foundation
 import UIKit
 
-public typealias LKShouldStartClosure = (UIWebView, URLRequest, UIWebViewNavigationType) -> Bool
+public typealias LKShouldStartClosure = (UIWebView, URLRequest, UIWebView.NavigationType) -> Bool
```
Comment (sberrevoets) [inline Sources/LambdaKit/UIWebView+LambdaKit.swift:28]: Since `UIWebView` is now formally deprecated we should probably just remove this file altogether

Comment (keith) [reply Sources/LambdaKit/UIWebView+LambdaKit.swift:28]: We need to change our internal stuff for this first

**Pull request merged**